### ISSUE TITLE
Doesn't work with preact

### ISSUE THREAD

**Issue by necolas** [kodyl/react-document-meta#30]
This line causes an error: https://github.com/kodyl/react-document-meta/blob/master/lib/utils.js#L6

**Comment by danieljuhl** [comment]
@necolas can you provide the full error message and a minimal reproducible sample?

**Comment by necolas** [comment]
```
Uncaught (in promise) TypeError: Converting circular structure to JSON(…)clone @ utils.js:25
reducePropsTostate @ index.js:37
emitChange @ index.js:47
componentWillMount @ index.js:88
setComponentProps @ preact.js:273
renderComponent @ preact.js:314
renderComponent @ preact.js:315
renderComponent @ preact.js:315
renderComponent @ preact.js:315
rerender @ preact.js:72
```

**Comment by danieljuhl** [comment]
Which props are you sending to DocumentMeta? Why are you giving it a circular structure? Children is the only prop, that would be circular - and it's excluded.

Can you please show a sample code, showing the props you are sending to DocumentMeta.

**Comment by necolas** [comment]
You can pass children-like content via any prop you want, e.g., `renderSidebar`.

**Comment by danieljuhl** [comment]
I know - but do you have a real use case? So that I can create a test and fix it :smile: 

Right now I can't figure out a case, where you would give document meta a react component as a prop

**Comment by necolas** [comment]
This is the object `clone` receives when using Preact

![image](https://cloud.githubusercontent.com/assets/239676/20541710/33b0b466-b0b3-11e6-8ad4-9201aec49762.png)

**Comment by danieljuhl** [comment]
Thanks! I'll look at it..

**Comment by danieljuhl** [comment]
@necolas I'm working on it right now.. I'll release/publish a new version later today.

**Comment by danieljuhl** [comment]
I've just released `v2.1.0` which should work with `preact`. Please let me know, if it isn't the case.
